src/demoserver.cpp

```diff
@@ -162,6 +162,23 @@ void DemoServer::handle_packet(AOPacket packet)
           load_demo(path);
           QString packet = "CT#DEMO#" + tr("Demo file loaded. Send /play or > in OOC to begin playback.") + "#1#%";
           client_sock->write(packet.toUtf8());
+          // Reset evidence list
+          client_sock->write("LE##%");
+          // Reset timers
+          client_sock->write("TI#0#3#0#%");
+          client_sock->write("TI#0#1#0#%");
+          client_sock->write("TI#1#1#0#%");
+          client_sock->write("TI#1#3#0#%");
+          client_sock->write("TI#2#1#0#%");
+          client_sock->write("TI#2#3#0#%");
+          client_sock->write("TI#3#1#0#%");
+          client_sock->write("TI#3#3#0#%");
+          client_sock->write("TI#4#1#0#%");
+          client_sock->write("TI#4#3#0#%");
+          // Set the BG to default (also breaks up the message queue)
+          client_sock->write("BN#default#wit#%");
+          // Stop the wait packet timer
+          timer->stop();
```

Comment:
Please move this to its own method thanks

Also if you want, we can totally modify the TI packet specs to add a command for resetting all timers.

Reply:
can u do it plz

Reply:
ur welcome :3